scripts/script.js

```diff
@@ -31,6 +31,18 @@ let misclickPenalty = 0;
 //Keep track of number of rabbits
 let numRabbits = 0;
 
+//Reset all the global variables
+function resetVariables() {
+  finalTime = 0;
+  avReactSpeed = 0;
+  clicked = false;
+  rabbitAppearTime = [];
+  rabbitClickedTimes = [];
+  rabbitPenalty = 0;
+  misclickPenalty = 0;
+  numRabbits = 0;
+}
+
 function clickPlayButton() {
   misclickPenalty = 0;
   numRabbits = 0;
```

Comment:
Do you have now nuRabbits = 0; on two places (line 43 and 48)? I think you can remove it from line 48